pkg/control/control.go

```diff
@@ -455,6 +455,7 @@ func (c *Controller) onNemesis(ctx context.Context, op *core.NemesisOperation) {
 	nemesis := core.GetNemesis(string(op.Type))
 	if nemesis == nil {
 		log.Errorf("nemesis %s is not registered", op.Type)
+		time.Sleep(30 * time.Second)
```

Comment:
Why need to wait 30s?

Reply:
Because the next round of `OnNemesis` starts immediately after current round returns, so a sleep is needed otherwise there will be a lot of logs.

Reply:
Make sense.